strapi/strapi
# fix(content-releases):disable edit and delete buttons in the Details page when you don't have permissions

### What does it do?

Adds the disable style to the Edit and Delete buttons in the Details page when a user doesn't have the permission to click them (before we just hide them)

### Why is it needed?

Because it is a better UX for the user to provide the buttons disabled

### How to test it?

- Open a new tab of the Admin and login as SuperAdmin and create a new user with a different Role
- Change the permission of the Role of the new user for the Content releases plugin (specifically Delete and Update permissions)
- Open a new tab of the Admin and login with the new user, and check if the buttons in the Details Release page are or not disabled
